src/x86/cpuid.c

```diff
@@ -692,6 +695,7 @@ struct topology* get_topology_info(struct cpuInfo* cpu, struct cache* cach, int
 
   switch(cpu->cpu_vendor) {
     case CPU_VENDOR_INTEL:
+    case CPU_VENDOR_TRANSMETA:
```

Comment:
I'm not sure of this. From what I recall, `get_topology_from_apic` is Intel only. It seems like it is getting the right number of cores but this is probably coming from `get_topology_from_udev`. Can you confirm? Just run with `--verbose` and it will show us how is it getting this information.

Reply:
You are right. Any help with finding out how to retrieve topology is appreciated. I believe we could also assume cores=1 on this platform.

```
[WARNING]: Can't read features information from cpuid (needed level is 0x00000007, max is 0x00000001)
[WARNING]: Can't read frequency information from cpuid (needed level is 0x00000016, max is 0x00000001). Using udev
[WARNING]: Can't read topology information from cpuid (needed level is 0x00000001, max is 0x00000001)
```